script/_network_manager

```diff
@@ -330,6 +330,12 @@ network_manager_install()
         return 0
     fi
 
+    if with DNS64; then
+        # bind9 provides DNS service
+        sudo sed -i 's/^#port=5353/port=0/g' /etc/dnsmasq.conf
+        sudo systemctl restart dnsmasq
```

Comment:
why do we install dnsmasq in the first place if `DNS64=1`?

Reply:
It provides other services in addition to DNS service.